app/views/cases/offender_sar/_subject_details_step.html.slim

```diff
@@ -25,7 +25,9 @@
   = f.radio_button_fieldset :subject_type,
     choices: Case::SAR::Offender::subject_types.keys
 
-  = f.text_area :subject_address, rows: '4'
+  = f.text_area :subject_address, rows: '4' 
+  .form-group
+    = link_to t('common.contacts.find_address'), '#', id: 'open-button'
```

Comment:
Can this be changed to a button, as there was a JAWS issue that made the link jump to the top of the page instead of opening the popup.